idl/cherami_server/store.thrift

```diff
@@ -227,6 +227,19 @@ struct RemoteReplicateExtentRequest {
   3: optional string extentUUID
 }
 
+// ListExtentsElem contains information to identify an extent on the store
+// - destinationUUID: is the uuid of the destination, when available
+// - extentUUID: is the uuid of the extent
+struct ListExtentsElem {
+  1: optional string destinationUUID
+  2: optional string extentUUID
```

Comment:
This is supposed to be a 'lightweight' API .. that quickly returns the list of extents without impacting IO, etc. We already have a "GetExtentInfo" API that can be used to query more detailed information about a given extent.